pages/blog.html

```diff
@@ -0,0 +1,377 @@
+            <p class="post__date">Luke Matthews l  November 8, 2021</p>
+          </div></a>
+          <a href="./blogPost.html" class="post__link">
+          <div class="post__item">
+            <img src="../images/woman-having-video-call-in-home-office-4240610.png" alt="Blog post" class="post__img">
+            <h5 class="post__title">The Five Main Benefits of Automating your Accounts Receivable Process</h5>
+            <p class="post__summary">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.</p>
+            <p class="post__date">Luke Matthews l  November 8, 2021</p>
+          </div></a>
+          <a href="./blogPost.html" class="post__link">
+           <div class="post__item">
+            <img src="../images/people-on-a-video-call-4226262.png" alt="Blog post" class="post__img">
+            <h5 class="post__title">How to Make Data-driven Pricing Decisions</h5>
+            <p class="post__summary">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.</p>
+            <p class="post__date">Luke Matthews l  November 8, 2021</p>
+          </div></a>
+          <a href="./blogPost.html" class="post__link">
+           <div class="post__item">
+            <img src="../images/close-up-photography-of-woman-sitting-beside-table-while-using-macbook-3178818.png" alt="Blog post" class="post__img">
+            <h5 class="post__title">How to Make Recurring Invoicing More Efficient</h5>
+            <p class="post__summary">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.</p>
+            <p class="post__date">Luke Matthews l  November 8, 2021</p>
+          </div></a>
+          <a href="./blogPost.html" class="post__link">
+           <div class="post__item">
+            <img src="../images/woman-writing-on-her-notebook-while-holding-a-cup-7616800.png" alt="Blog post" class="post__img">
+            <h5 class="post__title">How to Reduce Checkout Abandonment for more Conversions</h5>
+            <p class="post__summary">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.</p>
+            <p class="post__date">Luke Matthews l  November 8, 2021</p>
+          </div> </a>
+          <a href="./blogPost.html" class="post__link">
+          <div class="post__item">
+            <img src="../images/photo-of-female-engineer-working-on-her-workspace-3862632.png" alt="Blog post" class="post__img">
+            <h5 class="post__title">10 Customer Retention Strategies for better Dwell time</h5>
+            <p class="post__summary">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.</p>
+            <p class="post__date">Luke Matthews l  November 8, 2021</p>
+          </div></a>
+        </div>
+      </section>
+      <section class="cta">
```

Comment:
cta is not a descriptive name, you can name it like work or business, I suggest you change it.